addon/helpers/pikaday.js

```diff
@@ -11,57 +11,64 @@ deprecate(
 );
 
 const openDatepicker = function(element) {
-  $(element).click();
-
-  return PikadayInteractor;
+  const pickerElement = document.querySelector(element);
+  if (pickerElement) {
+    pickerElement.click();
+    return PikadayInteractor;
+  }
+  Ember.Logger.error(`${element} does not match any valid DOM element`);
```

Comment:
This is agreat idea to notify the user! I've read in an [RFC](https://emberjs.github.io/rfcs/0297-deprecate-ember-logger.html) that `Ember.Logger` will be deprecated at some point. Can we fail silently here? It begs the question of a more comprehensive error strategy, but I think that is outside the scope of this change.